package.json

```diff
@@ -8,11 +8,8 @@
   "homepage": "https://github.com/mongodb-js/mongosh",
   "author": "Compass Team <compass@mongodb.com>",
   "scripts": {
-    "prebootstrap-with-chromium": "npm install",
-    "bootstrap-with-chromium": "lerna bootstrap --concurrency=1",
-    "prebootstrap": "npm install",
-    "bootstrap": "cross-env PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=1 lerna bootstrap --concurrency=1",
-    "bootstrap-ci": "lerna bootstrap --ci",
```

Comment:
removed bootstrap-ci because bootstrapping in ci is complicated and I didn't want it split between this package.json, our evergreen yaml and a shell script. Rather just in one place as much as possible.